app/models/mediaflux/string_reverse.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+module Mediaflux
+  class StringReverse < Request
+    # This is only here to prove that we can use a mediaflux service that is provided by
+    # a java plugin.
+
+    # Constructor
+    # @param session_token [String] the API token for the authenticated session
+    def initialize(string:, session_token:)
+      super(session_token: session_token)
+      @string = string
+    end
+
+    # Specifies the Mediaflux service to use when querying Mediaflux.
+    # @return [String]
+    def self.service
+      "tigerdata.trivial :string #{@string}"
```

Comment:
The service should be just the name of the service: `tigerdata.trivial`, we pass the parameter as a separate block since Mediaflux expects them in a different part of the XML when it receives the request. 

Take a look at this for an example:  https://github.com/pulibrary/tigerdata-app/blob/main/app/models/mediaflux/asset_metadata_request.rb#L47-L53